test_data_e2e/generate_sha1sum_files.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+from scripts.checksum import compute_checksum
+
+
+def compute_checksum_in_folder(top_dir, algorithm: str):
+    """Traverse folder and write checksum file for each file found."""
+    for root, d_names, f_names in os.walk(top_dir):
+        for f_name in f_names:
+            if f_name.endswith(algorithm):
```

Comment:
Maybe good to skip hidden files too, like in [scan_data_checksum_files_pairs](https://github.com/thehyve/pmc-conversion/blob/0194a913e65d986826301d88ab0b18d512a18811/scripts/sync.py#L44) function.

Reply:
good point!